src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -335,11 +335,21 @@ def load_migration_products_offline(activations)
       def load_migrations_for_products(products, remote_product)
         log.info "Loading offline migrations for target product: #{remote_product.inspect}"
         log.info "Installed products: #{products.inspect}"
+
         self.migrations = registration_ui.offline_migration_products(products, remote_product)
 
         if migrations.empty?
-          # TRANSLATORS: Error message
-          Yast::Report.Error(_("No migration product found."))
+          msg = [
+            # TRANSLATORS. Error message
+            _("No migration product found."),
+            # TRANSLATORS: Help message
+            _("Please, boot the original system and make sure " \
+              "that it and all registerable products are correctly registered.\n" \
+              "Also check that the installed system is supported for upgrade to this new product.")
```

Comment:
Maybe we could use the real product name and replace the generic "this new product" words.

```suggestion
              "Also check that the installed system is supported for upgrade to \n%{product}.") % {product: remote_product}
```